web/server/codechecker_server/database/config_db_model.py

```diff
@@ -144,6 +144,47 @@ def __init__(self, token, user_name, groups, description=None,
         self.last_access = datetime.now()
 
 
+class PersonalAccessToken(Base):
+    __tablename__ = 'personal_access_tokens'
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+
+    user_name = Column(String)
+
+    token_name = Column(String)
+    token = Column(CHAR(32), nullable=False, unique=True)
+
+    description = Column(String)
+
+    last_access = Column(DateTime, nullable=False)
+    expiration = Column(DateTime)
+
+    auth_session_id = Column(
+        Integer,
+        ForeignKey('auth_sessions.id', deferrable=False, ondelete='CASCADE'),
+        nullable=False)
+
+    __table_args__ = (
+        UniqueConstraint('user_name', 'token_name'),
+    )
+
+    def __init__(
+        self,
+        user_name,
+        token_name,
+        token,
+        description,
+        auth_session_id
+    ):
+        self.user_name = user_name
+        self.token_name = token_name
+        self.token = token
+        self.description = description
+        self.auth_session_id = auth_session_id
+        self.last_access = datetime.now()
+        self.expiration = self.last_access + timedelta(days=365)
```

Comment:
This default expiration should be configurable, and ideally, user-selectable (with a configurable upper limit).

Reply:
Will be done later.